components/classes/healthcare-plan.schema.json

```diff
@@ -0,0 +1,61 @@
+{
+  "meta:license": [
+    "Copyright 2022 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/classes/healthcare-plan",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Healthcare Plan",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:tags": {
+    "industry": ["healthcare_services"]
+  },
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "description": "Details of a Health Insurance product/plan provided by an organization.",
+  "definitions": {
+    "plan": {
+      "properties": {
+        "xdm:planID": {
+          "Title": "Plan ID",
+          "type": "string",
+          "description": "Unique identifier of the insurance plan."
+        },
+        "xdm:planName": {
+          "Title": "Plan Name",
+          "type": "string",
+          "description": "Name of the insurance plan."
+        },
+        "xdm:planType": {
```

Comment:
If its only for primary, secondary kind of healthcare plans, let's move this to the healthcare-plan-details FG. We can also leave it here with no industry specific enums and add "isPrimary", isSecondary" and "isTeritiary" properties in the Field group for healthcare plan details.